qiskit/version.py

```diff
@@ -101,7 +102,10 @@ def __init__(self):
         self._loaded = False
 
     def _load_versions(self):
-        import pkg_resources
+        if sys.version_info >= (3, 8):
+            from importlib.metadata import version
+        else:
+            from importlib_metadata import version
```

Comment:
FWIW, I think we can probably do this as a module level import. The `pkg_resources` import was done at runtime because it was exceedingly slow. I don't expect `importlib.metadata` to have this issue (but it would be worth checking to confirm) and I think it probably gets pulled in already via our plugin interfaces. That being said it doesn't actually matter in practice so it's not worth a follow up PR.

Reply:
Thanks. I did not know the reason of lazy import here.